### PR TITLE
fix: add automatic releases

### PR DESCRIPTION
**Pull request by wtrocki**
CC @jschuler

### CODE REVIEW

File: .github/workflows/npm-publish.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://npm.pkg.github.com/
+      - run: yarn build
+      - run: npm publish
+        env:
+          NODE_AUTH_TOKEN: ${{secrets.GITHUB_TOKEN}}
```
Comment (wtrocki) [inline .github/workflows/npm-publish.yml:46]: This requires auth token to be provided in github secrets. Also this is not automatic.
Process is simple:

Bump packages in the repositories.
Create github release

Comment (jschuler) [reply .github/workflows/npm-publish.yml:46]: This is awesome @wtrocki ! So I will add `GITHUB_TOKEN` and `npm_token` secrets under repo settings right? anything else I need to do to hook this up?

File: .github/workflows/npm-publish.yml [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    needs: build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 12
+          registry-url: https://registry.npmjs.org/
+      - run: yarn build
+      - run: npm publish
```
Comment (jschuler) [inline .github/workflows/npm-publish.yml]: should this also be changed to cd into packages/module first?

Comment (wtrocki) [reply .github/workflows/npm-publish.yml]: Yes. Sorry. I did not quite get why this is the only module we publish? is that right?
```suggestion
      - run: cd packages/module && npm publish
```

File: .github/workflows/npm-publish.yml (hunk 1 above)
Comment (wtrocki) [reply .github/workflows/npm-publish.yml:46]: That is it really. Only those tokens are needed.

File: .github/workflows/npm-publish.yml (hunk 2 above)
Comment (jschuler) [reply .github/workflows/npm-publish.yml]: yes, currently it's the only module that is published

**Comment by wtrocki** [comment]
@jschuler Also since this is on the fork you might need to bring that into the repository for testing builds. If tokens are there this must work

**Comment by jschuler** [comment]
Thank you @wtrocki let's get this merged, if something needs changing afterwords we can do that

**Pull request merged**